File: Basics of C Programming/factorial_using_recursion.c
#include <stdio.h>

int factorial(int a){
    if (a==1){
        return 1;
    }else{
        return a*factorial(a-1);
    }
}

int main(){
    int a;
    printf("Enter the number:");
    scanf("%d",&a);
    int b=factorial(a);
    printf("Factorial of the number %d is %d",a,b);
    return 0;
}

File: python/ATP record/factorial_using_recursion.py
def factorial(a):
    if a==1:
        return 1
    else:
        return a*factorial(a-1)
    
def numinput():
    a=int(input("enter the number:"))
    return a

a=numinput()
b=factorial(a)
print("The factorial of",a,"is",b)
